CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 ## 3.34.0 (Unreleased)
+NOTES:
+* redis: explicitly noted in `google_redis_instance` documentation that `"REDIS_5_0"` is supported ([#6917](https://github.com/terraform-providers/terraform-provider-google/pull/6917))
+
+IMPROVEMENTS:
+* bigtable: added support for labels in `google_bigtable_instance` ([#6921](https://github.com/terraform-providers/terraform-provider-google/pull/6921))
+* cloudfunctions: updated `google_cloudfunctions_function` datasource to include new fields. ([#6935](https://github.com/terraform-providers/terraform-provider-google/pull/6935))
+* redis: added `persistence_iam_identity` output field to `google_redis_instance` ([#6917](https://github.com/terraform-providers/terraform-provider-google/pull/6917))
+* storage: added google_storage_bucket_object.media_link. ([#6897](https://github.com/terraform-providers/terraform-provider-google/pull/6897))
+
+BUG FIXES:
+* all: fixed markdown for enum values ([#6924](https://github.com/terraform-providers/terraform-provider-google/pull/6924))
```

Comment:
I don't usually bother with release notes for website-only changes, but I don't care enough to say not to include this. Maybe just say something about it being a website change?